docs/UserGuide.md

```diff
@@ -1,192 +1,147 @@
----
```

Comment:
I think we should consider keeping most of the previous structure first for v1.1, before implementing our own formatting?